### PR TITLE
Fix VERIFICATION & Add Chocolatey package as AppVeyor artifact

### PR DESCRIPTION
**Pull request by tomkerkhove**
Related to #137.

Expected output in AppVeyor - https://ci.appveyor.com/project/tomkerkhove/servicebusexplorer
However, need to check why it doesn't find any artifacts anymore.

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Feel free to squash when merging in, a bit too many changes.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.103-PullRequest0143.14.build.288 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.103-PullRequest0143.14.build.288) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/8a7f5c1ccb by @tomkerkhove)

File: VERIFICATION.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (SeanFeldman) [inline VERIFICATION.txt]: Do we need to have this file at all? Could `Generate-VerificationFile.ps1` generate it as it's only needed during that script's execution _and_ `$OutputFilePath` is passed in?

Comment (tomkerkhove) [inline VERIFICATION.txt]: That's right, that's why I removed it

Comment (SeanFeldman) [inline VERIFICATION.txt]: 😄 👍

**Comment by SeanFeldman** [comment]
@tomkerkhove have you submitted with the same version? Seems like this is what choco admins want.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.103-PullRequest0143.15.build.289 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.103-PullRequest0143.15.build.289) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/fae9c9da2e by @tomkerkhove)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.103-PullRequest0143.16.build.290 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.103-PullRequest0143.16.build.290) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/121b6ee45c by @tomkerkhove)

**Comment by tomkerkhove** [comment]
@SeanFeldman Please re-review. Fixed the issue with AppVeyor and hardcoded the version for choco validation to pass first

**Comment by tomkerkhove** [comment]
@SeanFeldman Rolled back the hardcoded changes and will create new version of Chocolatey package once it's merged in

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.0.103-PullRequest0143.18.build.291 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/build/4.0.103-PullRequest0143.18.build.291) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/54024784be by @tomkerkhove)

**Comment by SeanFeldman** [comment]
@tomkerkhove Squash?

**Comment by tomkerkhove** [comment]
@seanfeldman I think it's best to squash merge indeed

**Pull request merged**